internal/action/actions.go

```diff
@@ -1696,31 +1696,26 @@ func (h *BufPane) SelectPageDown() bool {
 	return true
 }
 
-// CursorPageUp places the cursor a page up
+// CursorPageUp scrolls the view up a page,
```

Comment:
This description emphasizes the "view" aspect so much that it forgets that first of all it is the cursor that moves (that is what these action do: they move the cursor a page up or down, right?), and the view moves as a result of that, not the other way around.

Reply:
Updated.